### PR TITLE
Astro 1/10 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -23,6 +23,23 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+### New ways to build data pipelines in the Cloud IDE
+
+The Astro Cloud IDE now includes updated cell types for building your data pipelines. The new and updated cell types are:
+
+- **SQL**: Runs a SQL query against a provided database connection and persists the results of the query in an XCom for use by other cells.
+- **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse.
+- **Markdown**: Adds documentation to your generated DAG code.
+
+For more information, see [Run SQL](cloud-ide/run-sql.md) and [Document your pipeline](cloud-ide/document-pipeline.md).
```
Comment (jwitz) [inline astro/release-notes.md:36]: See https://github.com/astronomer/docs/pull/1722 for drafts of these docs.

File: astro/release-notes.md [hunk 2]
```diff
@@ -23,6 +23,23 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.8.4 ([Release notes](cli/release-notes.md))
 
+## January 10, 2022
```
Comment (kwcanuck) [inline astro/release-notes.md:26]:
```suggestion
## January 10, 2023 
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -23,6 +23,23 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.8.4 ([Release notes](cli/release-notes.md))
 
+## January 10, 2022 
+
+### New ways to build data pipelines in the Cloud IDE
+
+The Astro Cloud IDE now includes updated cell types for building your data pipelines. The new and updated cell types are:
```
Comment (kwcanuck) [inline astro/release-notes.md:30]:
```suggestion
The following cell types have been added to the Astro Cloud IDE:
```

File: astro/release-notes.md [hunk 4]
```diff
@@ -23,6 +23,23 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+- **SQL**: Runs a SQL query against a provided database connection and persists the results of the query in an XCom for use by other cells.
```
Comment (kwcanuck) [inline astro/release-notes.md:32]:
```suggestion
- **SQL**: Runs a SQL query against a defined database connection and persists the results of the query in an XCom file for use by other cells.
```

File: astro/release-notes.md [hunk 5]
```diff
@@ -23,6 +23,23 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse.
```
Comment (kwcanuck) [inline astro/release-notes.md:33]:
```suggestion
- **Warehouse SQL**: Runs a SQL query against a defined database connection and stores the results in your data warehouse.
```

**Review by kwcanuck — APPROVED** [review]

File: astro/release-notes.md [hunk 6]
```diff
@@ -23,6 +23,23 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
 **Latest CLI Version**: 1.8.4 ([Release notes](cli/release-notes.md))
 
+## January 10, 2023 
+
+### New Astro Cloud IDE cell types
+
+The following cell types have been added to the Astro Cloud IDE:
+
+- **SQL**: Runs a SQL query against a defined database connection and persists the results of the query in an XCom file for use by other cells.
+- **Warehouse SQL**: Runs a SQL query against a defined database connection and stores the results in your data warehouse.
```
Comment (jwitz) [inline astro/release-notes.md:33]:
```suggestion
- **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse.
```

File: astro/release-notes.md [hunk 7]
```diff
@@ -23,6 +23,28 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.8.4 ([Release notes](cli/release-notes.md))
 
+## January 10, 2023 
+
+### New Astro Cloud IDE cell types
+
+The following cell types have been added to the Astro Cloud IDE:
```
Comment (paolaperaza) [inline astro/release-notes.md:30]:
```suggestion
In the Cloud IDE, you can now create tasks with the following cell types:
```

File: astro/release-notes.md [hunk 8]
```diff
@@ -23,6 +23,28 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+- **SQL**: Runs a SQL query against a defined database connection and persists the results of the query in an XCom file for use by other cells. Use this cell type to run smaller queries and store the results in Airflow for quick access in other cells.
+- **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse. Use this cell type for data operations that require more storage and reliability.
```
Comment (paolaperaza) [inline astro/release-notes.md:33]:
```suggestion
- **Warehouse SQL**: Runs a SQL query against an existing database connection and stores the results in your data warehouse. Use this cell type for data operations that require more storage and a higher level of reliability.
```

File: astro/release-notes.md [hunk 9]
```diff
@@ -23,6 +23,28 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.8.4 ([Release notes](cli/release-notes.md))
 
+## January 10, 2023 
+
+### New Astro Cloud IDE cell types
+
+The following cell types have been added to the Astro Cloud IDE:
+
+- **SQL**: Runs a SQL query against a defined database connection and persists the results of the query in an XCom file for use by other cells. Use this cell type to run smaller queries and store the results in Airflow for quick access in other cells.
```
Comment (paolaperaza) [inline astro/release-notes.md:32]:
```suggestion
- **SQL**: Runs a SQL query against an existing database connection and persists the results of the query in an XCom file for use by other cells. Use this cell type to run smaller queries and store the results in Airflow for quick access in other cells.
```

File: astro/release-notes.md [hunk 10]
```diff
@@ -23,6 +23,28 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse. Use this cell type for data operations that require more storage and reliability.
+- **Markdown**: Adds documentation to your generated DAG code. Use this cell to make it easier for team members to collaborate on the same pipeline.
```
Comment (paolaperaza) [inline astro/release-notes.md:34]:
```suggestion
- **Markdown**: Adds documentation as in-line comments to your generated DAG code. Use this cell to make it easier for team members to collaborate on the same pipeline.
```

File: astro/release-notes.md [hunk 11]
```diff
@@ -23,6 +23,28 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+For more information, see [Run SQL](cloud-ide/run-sql.md) and [Document your pipeline](cloud-ide/document-pipeline.md).
+
+### Additional improvements
+
+- The default value for `AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL` has been reduced from 5 minutes to 30 seconds for all Deployments regardless of Runtime version.
```
Comment (paolaperaza) [inline astro/release-notes.md:40]:
```suggestion
- To reduce the time it takes for Airflow to parse new DAG files, the default value for `AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL` has been reduced from 5 minutes to 30 seconds for all Deployments regardless of Runtime version. For most users, this means that you will see new DAGs appear in the Airflow UI faster.
```

**Review by paolaperaza — APPROVED** [review]
Coupla comments but this LGTM @jwitz

File: astro/release-notes.md (hunk 7 above)
Comment (jwitz) [reply astro/release-notes.md:30]: This isn't quite accurate, because the Markdown cell isn't task-based

File: astro/release-notes.md [hunk 12]
```diff
@@ -23,6 +23,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+## January 10, 2023 
+
+### New Astro Cloud IDE cell types
+
+The following cell types have been added to the Astro Cloud IDE:
+
+- **SQL**: Runs a SQL query against an existing database connection and persists the results of the query in an XCom file for use by other cells. Use this cell type to run smaller queries and store the results in Airflow for quick access in other cells.
+- **Warehouse SQL**: Runs a SQL query against an existing database connection and stores the results in your data warehouse. Use this cell type for data operations that require more storage and a higher level of reliability.
+- **Markdown**: Adds documentation as in-line comments to your generated DAG code. Use this cell to make it easier for team members to collaborate on the same pipeline.
+
```
Comment (jwitz) [inline astro/release-notes.md:35]:
```suggestion
```

**Pull request merged**